doc/usecases/layers/plants_layer.md

```diff
@@ -2,36 +2,36 @@
-  - The user is able to add plants and is presented with a list of seasonal plants (for the selected time of the year), in following groups:
-    - available seeds with early expiration days first
-    - diversity constraints: suggestions which not-yet-used plants fit next to the plant just added, or at a place selected (spatial and temporal)
-    - which were recently planted
-    - and lastly favourites of users
-  - The user gets a visual indication which parts of the map are better suited for the plant to be placed (e.g. zones, pH value, moisture, animals or shadows).
-  - The user positions the plant element in the appropriate location in the map.
-  - When drawing an area of plants, the size of the arena and the number of plants is shown next to the mouse.
-  - The user is able to view the relationships between the plants by looking at the lines connecting them.
-    The lines indicate the strength of the relationship.
-  - The user is able to move, edit (e.g. when planted, when harvested) and delete (either that it never existed or that it was removed from the garden) selected plants.
-  - The user adjusts the plant elements and their relationships as needed.
-- **Alternative scenario:**
-  - The user accidentally adds or moves a plant element in the wrong location and uses the app's undo function to correct the mistake.
-- **Error scenario:**
-  - The user attempts to add, move or edit a plant element but the app is experiencing technical difficulties and is unable to complete the request, displaying an error message.
-  - There is an error in the app's plant relationship indication and the lines connecting the plants are not displayed correctly. In this case, the app displays an error message.
-- **Postcondition:**
-  The user's map includes the added, edited, moved or deleted plant element as desired.
-- **Non-functional Constraints:**
-  - Performance
-  - Partial offline availability
+-   **Precondition:**
+    The user has opened the app and has selected the plants layer.
+-   **Main success scenario:**
+    -   The user is able to add plants and is presented with a list of seasonal plants (for the selected time of the year), in following groups:
+        -   available seeds with early expiration days first
+        -   diversity constraints: suggestions which not-yet-used plants fit next to the plant just added, or at a place selected (spatial and temporal)
+        -   which were recently planted
+        -   and lastly favourites of users
+    -   The user gets a visual indication which parts of the map are better suited for the plant to be placed (e.g. zones, pH value, moisture, animals or shadows).
+    -   The user positions the plant element in the appropriate location in the map.
+    -   When drawing an area of plants, the size of the arena and the number of plants is shown next to the mouse.
+    -   The user is able to view the relationships between the plants by looking at the lines connecting them.
```

Comment:
Let's specify the "lines" between plants mentioned in this use case i.e. how they should be presented.

There are the following cases that I see right now:

1. If the lines are drawn just on top of the map between plants while the plants layer is enabled
2. If the lines are drawn after the user clicks a special button to indicate the relationships, lines are also drawn on top of the map.
3. After the user clicks on a specific plant to indicate the relationships relative to the chosen one

Could you please share how do you see this feature? Let's update the description here

@markus2330 
@buenaflor

Reply:
Excellent questions, I created #95 for clarifications. Please comment there.

>  If the lines are drawn just on top of the map between plants while the plants layer is enabled

They are only drawn while drag&drop/move the plant, which is only possible if you are in the plant layer, then the lines get added to the warnings layer.

> If the lines are drawn after the user clicks a special button to indicate the relationships, lines are also drawn on top of the map.

This "special button" is the visibility of the warnings layer.

> After the user clicks on a specific plant to indicate the relationships relative to the chosen one

Yes, while moving plants, the relationships etc. should be shown again.